### PR TITLE
feat(ui/notWorkingButtonDelay): button appears 10 secs after QR is loads

### PR DESCRIPTION
**Pull request by KyuubiNoKitsune**

### CODE REVIEW

File: app/javascript/components/payment-error-button.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      this.$modal.hide('apology-modal');
+    },
+  },
+  mounted() {
+    setTimeout(() => {
+      this.showApologyButton = true;
+    }, SHOW_APOLOGY_BUTTON);
+  },
+}
+</script>
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Newline required at end of file but not found                              eol-last

File: app/javascript/components/payment-error-button.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    hide() {
+      this.$modal.hide('apology-modal');
+    },
+  },
+  mounted() {
+    setTimeout(() => {
+      this.showApologyButton = true;
+    }, SHOW_APOLOGY_BUTTON);
+  },
+}
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Missing semicolon                                                          semi

File: app/javascript/components/payment-error-button.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          ¿No funciona?
+        </button>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import { mapState, mapActions, mapGetters } from 'vuex';
+import invoiceApi from '../api/invoices';
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected empty line after import statement not followed by another import  import/newline-after-import

File: app/javascript/components/payment-error-button.vue [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        >
+          ¿No funciona?
+        </button>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import { mapState, mapActions, mapGetters } from 'vuex';
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: 'mapState' is defined but never used                                       no-unused-vars<br>'mapActions' is defined but never used                                     no-unused-vars<br>'mapGetters' is defined but never used                                     no-unused-vars

File: app/javascript/components/payment-error-button.vue [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
+          @click="show"
+        >
+          ¿No funciona?
+        </button>
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 6 spaces but found 8 spaces                        vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
+          @click="show"
+        >
+          ¿No funciona?
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
+          @click="show"
+        >
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 6 spaces but found 8 spaces                        vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
+          @click="show"
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    <modal name="apology-modal">
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  <div class="payment-error-button">
+    <modal name="apology-modal">
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 8 spaces but found 10 spaces                       vue/html-indent<br>Attribute "if" should be on a new line                                     vue/max-attributes-per-line

File: app/javascript/app.vue [hunk 11]
```diff
@@ -88,6 +88,7 @@ export default {
     },
   },
   mounted() {
+    console.log('APP COMPONENT');
```
Comment (dacasas) [inline app/javascript/app.vue]: Se te pasó este o no?

File: app/javascript/components/app-invoice.vue [hunk 12]
```diff
@@ -46,31 +40,23 @@
...
-      <div class="invoice__error-btn ">
-        <button
-          type="button"
-          class="btn"
-          @click="show"
-        >
-          ¿No funciona?
-        </button>
-      </div>
+        <errorNotification v-if="loading" />
```
Comment (dacasas) [inline app/javascript/components/app-invoice.vue]: este no debería ser `!loading`?, para que salga cuando ya se dejó de cargar.

File: app/javascript/components/payment-error-button.vue [hunk 13]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="payment-error-button">
+    <modal name="apology-modal">
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 6 spaces but found 8 spaces  vue/html-indent

File: app/javascript/components/app-invoice.vue [hunk 14]
```diff
@@ -46,31 +40,23 @@
...
     </div>
   </div>
 </template>
 
 <script>
 import { mapState, mapActions, mapGetters } from 'vuex';
 import loading from './loading.vue';
-import invoiceApi from '../api/invoices';
+import errorNotification from './payment-error-button.vue'
```
Comment (monkeyci) [inline app/javascript/components/app-invoice.vue]: Missing semicolon                                    semi

File: app/javascript/components/app-invoice.vue (hunk 12 above)
Comment (monkeyci) [inline app/javascript/components/app-invoice.vue]: Expected indentation of 6 spaces but found 8 spaces  vue/html-indent

**Pull request merged**